### DISCUSSION TITLE
Text widgets

### DISCUSSION THREAD

**Discussion by ctuxboy** [BoidCMS/BoidCMS#23] [Ideas]
Hello,
Have you planned for adding text-widget on pages/posts? It would be more flexibility using BoidCMS.

**Comment by sysadev** [comment]
I haven't think of that 😕, but it sounds pretty cool.
Any idea how it should be implemented?

**Reply by ctuxboy** [reply]
Similar like in WonderCMS? i don't if it can implemented, BoidCMS using another way for handle pages.

**Reply by sysadev** [reply]
You mean like WonderCMS's blocks?

**Reply by ctuxboy** [reply]
Yep

**Reply by ctuxboy** [reply]
Or maybe adding custom fields option similar like in WordPress? Maybe this is a better way adding custom fields tot the create/update page form?

**Reply by sysadev** [reply]
Yeah, it'll be better.

But, do you think that the custom fields should be page specific?

**Reply by ctuxboy** [reply]
maybe you can link it with a custom php-template?
Or another way empliment custom fields, like the way how the flatfile cms [Automad](https://automad.org/) handle custom fields?

**Reply by sysadev** [reply]
Sure, thanks

**Reply by ctuxboy** [reply]
NP ;)
